ansible/deploy-postgres-11.yml

```diff
@@ -0,0 +1,43 @@
+---
+
+# Deploy PostgreSQL 11.7
+
+- import_playbook: ansible-version.yml
+
+- hosts: ams-pg.ooni.nu
+  gather_facts: true # to gather `ansible_service_mgr`
+  tags: postgresql11
+  roles:
+    - role: postgresql11
+
+
+      # etckeeper
+      # netdata 
+      # firewall
+
+#- name: install prometheus-postgres-exporter
```

Comment:
I think you need to nest tasks inside of the task key instead of being at the role level. It's probably better though to put these inside a dedicated role.

Reply:
Yes, the comments were just temporary TODOs. Removed